main/pipeline/pipeline.h

```diff
@@ -48,6 +48,21 @@ size_t partitionPackageFiles(const core::GlobalState &gs, absl::Span<core::FileR
 void unpartitionPackageFiles(std::vector<ast::ParsedFile> &packageFiles,
                              std::vector<ast::ParsedFile> &&nonPackageFiles);
 
+struct CondensationLayerInfo {
+    // The `__package.rb` sources in this layer of the condensation graph.
+    absl::Span<ast::ParsedFile> packageFiles;
+
+    // The ruby sources in this layer of the condensation graph.
```

Comment:
There are RBI files in there too, not sure if it's worth clarifying that ("Ruby source" typically only refers to `*.rb` files)

<img width="628" height="721" alt="Image" src="https://github.com/user-attachments/assets/e536a6c7-ff21-4438-96cd-65762dbedc93" />

Reply:
I'll fix the comment to clarify that this includes RBIs.